### ISSUE TITLE
Report dependency class as unknown, even if it's in vendor folder

### ISSUE THREAD

**Issue by keradus** [maglnet/ComposerRequireChecker#20]
Hi !
I would be grateful for any hints on that !
`Symfony\Component\Finder\Finder` is reported as unknown, even if it's properly installed as dependency

```
ker@dus:~/github/FriendsOfPHP/PHP-CS-Fixer$ composer update -q

ker@dus:~/github/FriendsOfPHP/PHP-CS-Fixer$ php composer-require-checker.phar check | grep Finder
| Symfony\Component\Finder\Finder                                     |                    |
| Symfony\Component\Finder\SplFileInfo                                |                    |
| Symfony\Component\Process\PhpExecutableFinder                       |                    |

ker@dus:~/github/FriendsOfPHP/PHP-CS-Fixer$ cat vendor/symfony/finder/Finder.php | grep -P "(?:class|namespace) "
namespace Symfony\Component\Finder;
class Finder implements \IteratorAggregate, \Countable
```

**Comment by Ocramius** [comment]
That's because they're not in your root composer.json dependencies, yet you
are using it.

**Comment by keradus** [comment]
sticking to Finder example... it is:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.3.1/composer.json#L24

**Comment by Ocramius** [comment]
Interesting: needs investigation then.

**Comment by maglnet** [comment]
Could you please check the latest release:
https://github.com/maglnet/ComposerRequireChecker/releases/tag/0.1.3

There seemed to be a problem finding the correct path when used as a `.phar` file.

**Comment by keradus** [comment]
tested with 0.1.4 already. now it works, thanks ;)
